GVFS/GVFS.Platform.Windows/WindowsPlatform.cs

```diff
@@ -325,6 +325,14 @@ public override bool TryGetGVFSEnlistmentRoot(string directory, out string enlis
             return WindowsPlatform.TryGetGVFSEnlistmentRootImplementation(directory, out enlistmentRoot, out errorMessage);
         }
 
+        public override bool TryGetDefaultLocalCacheRoot(string enlistmentRoot, out string localCacheRoot, out string localCacheRootError)
+        {
+            string pathRoot = Path.GetPathRoot(enlistmentRoot);
```

Comment:
Should the `Path.GetPathRoot` and `Path.Combine` calls be in a  `try { } catch (ArgumentException e) { }` block?

Reply:
That's a good idea.  We weren't doing that before, but I think it's better to be consistent with the POSIX implementation.